doc/src/manual/constructors.md

```diff
@@ -573,6 +574,7 @@ Closest candidates are:
 ```
 
 This constructor will be invoked by the syntax `SummedArray(a)`. The syntax `new{T,S}` allows
-specifying parameters for the type to be constructed, i.e. this call will return a `SummedArray{T,S}`.
-`new{T,S}` can be used in any constructor definition, but for convenience the parameters
-to `new{}` are automatically derived from the type being constructed when possible.
+specifying parameters for the type to be constructed and this call will return a
```

Comment:
The former "i.e." strikes me as more appropriate than the "and"; perhaps revert?